pallets/parachain-staking/src/lib.rs

```diff
@@ -1532,6 +1532,9 @@ pub mod pallet {
 					// clean up storage items that we no longer need
 					<DelayedPayouts<T>>::remove(paid_for_round);
 					<Points<T>>::remove(paid_for_round);
+					// remove any AtStake entries that are left over. this will happen any time a
+					// selected collator produces no blocks for the entire round.
+					<AtStake<T>>::iter_prefix(paid_for_round).drain().collect();
```

Comment:
We should use `remove_prefix` here to avoid reading the value in PoV.
https://crates.parity.io/frame_support/storage/types/struct.StorageDoubleMap.html#method.remove_prefix

```suggestion
					<AtStake<T>>::remove_prefix(paid_for_round, None);
```

Reply:
Actually I included this fix in https://github.com/PureStake/moonbeam/pull/1878 since it's a pre-requisite for the migration.